var/spack/repos/builtin/packages/py-fenics-ffcx/package.py

```diff
@@ -17,14 +17,14 @@ class PyFenicsFfcx(PythonPackage):
     version('main', branch='main')
     version('0.1.0', sha256='98a47906146ac892fb4a358e04cbfd04066f12d0a4cdb505a6b08ff0b1a17e89')
 
-    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('python@3.7:', type=('build', 'run'))
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-cffi', type=('build', 'run'))
+    depends_on('py-cffi', type=('run'))
```

Comment:
I think `cffi` is also needed when you build the Python extension, right?

Reply:
No, it's used when the Python module does just-in-time compilation at runtime and returns a compiled module to the user.